src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.ts

```diff
@@ -226,6 +227,8 @@ export class BadgeClassEditFormComponent extends BaseAuthenticatedRoutableCompon
 				// but since it doesn't make sense to remove the
```

Comment:
The comments belong to the control below the ones you created. Also, is the `studyLoad` control actually still used, despite there being the new hours and minutes controls? Also, shouldn't at least one of the two new fields also be required to be a positive integer?

Reply:
1. ´studyload` control is not needed anymore. i deleted it alongside the comment 
2. i cant quite grasp your second question,  i added the `positiveIntegerOrNull()` validation, no?

Reply:
@Thiemann96 yes, but that also allows 0. So currently it's possible to enter 0 for both fields, whereas before a duration of 0 was not allowed. I don't think it's entirely trivial though, because at least one field would have to be non-0, for which we don't have a validator yet.

Reply:
Ah yeah of course, good catch. It is indeed not as trivial as I hoped, especially with dynamically added input fields for competencies.